action.yml

```diff
@@ -42,6 +46,16 @@ runs:
         exit 1
       fi
       echo "::set-output name=image::$IMAGE_NAME"
+  
+  - id: generate-cache-file
+    shell: bash
+    run: |
+      if [[ ${{ inputs.cache }} == true ]]; then
+        CACHE_FILE="/tmp/.buildx-cache"
+      else
+        CACHE_FILE="/tmp/throwaway"
+      fi
```

Comment:
I am not sure i understand the purpose of this logic and the `/tmp/throwaway` location if the `cache` input is not enabled.  Does this path mean anything semantically?  In other words, what is the difference between setting `CACHE_FILE="/tmp/throwaway"` when cache is disabled vs just leaving it as `CACHE_FILE="/tmp/.buildx-cache"` regardless?

Reply:
IIRC, the idea here was to workaround an inability to wrap the `cache-to` option in the docker build in an if statement: if the user specifies they don't want a cache, we dump the docker cache into a dummy file that isn't picked up by the docker caching mechanism